Queueomatic/Server/Endpoints/Participant/Add/AddNewParticipantEndpoint.cs

```diff
@@ -1,32 +1,52 @@
-﻿using FastEndpoints;
+﻿using System.Security.Claims;
+using FastEndpoints;
+using FastEndpoints.Security;
+using Queueomatic.DataAccess.Models;
+using Queueomatic.Server.Services.ParticipantService;
 
 namespace Queueomatic.Server.Endpoints.Participant.Add;
 
-public class AddNewParticipantEndpoint: Endpoint<AddNewParticipantRequest, AddNewParticipantResponse>
+public class AddNewParticipantEndpoint: Endpoint<AddNewParticipantRequest>
 {
+    private readonly IConfiguration _configuration;
+    private readonly IParticipantService _participantService;
+
+    public AddNewParticipantEndpoint(IConfiguration configuration, IParticipantService participantService)
+    {
+        _configuration = configuration;
+        _participantService = participantService;
+    }
+
     public override void Configure()
     {
         Verbs(Http.POST);
         Routes("/rooms/{roomId}/newParticipant");
```

Comment:
Maybe clump verbs and routes into a single one?

Reply:
Forgot to shrink it.